### PR TITLE
sort pages for a resource

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

File: spec/page_client_spec.rb [hunk 1]
```diff
@@ -53,14 +53,15 @@
...
-                     resource_type_id: 1,
-                     system_id: 1)
+    resource = Resource.create!(abbreviation: 'test',
+                                onesky_project_id: 1,
+                                name: name,
+                                description: description,
+                                resource_attributes: attributes,
+                                resource_type_id: 1,
+                                system_id: 1)
+    resource.pages = pages
```
Comment (dsgoers) [inline spec/page_client_spec.rb]: Why did you need to do this?

Comment (frett) [reply spec/page_client_spec.rb]: it was some sort of weird initialization cycle bug. when the resource xml validation rule ran the resource property on the page was nil.

By assigning the pages afterwards the resource property is correctly set

Comment (frett) [reply spec/page_client_spec.rb]: I'm guessing having the closure defined for pages causes it to be handled slightly differently within AR.

**Comment by frett** [comment]
I'm currently testing on stage

**Comment by frett** [comment]
it's ordering correctly and passing the spec on stage, I'm going to merge :)
